oqs/src/sig.rs

```diff
@@ -306,11 +343,12 @@ impl Sig {
     ) -> Result<()> {
         let signature = signature.into();
         let pk = pk.into();
+        if signature.bytes.len() > self.length_signature()
+            || pk.bytes.len() > self.length_public_key()
```

Comment:
```suggestion
            || pk.bytes.len() != self.length_public_key()
```
Match existing functionality